test/server_logs_test.go

```diff
@@ -153,3 +154,44 @@ func changeLogLevel(l string) string {
 
 	return "INFO"
 }
+
+// waitForHealthT returns cluster health when environment is healthy.
```

Comment:
we have already similar method: https://github.com/arangodb/go-driver/blob/0af2b60ac232c1a0a25abd67799394a9233f4919/test/client_test.go#L371-L371

Could you consider merging them?

Reply:
changed